tests/review-pr-consolidate.test.sh

```
@@ -0,0 +1,889 @@
+    "RCXCa: a recorded invoking PR is returned without asking gh — so it survives HEAD moving to the combine branch"
+
+  # Fail CLOSED when there is nothing recorded and nothing resolvable: the old
+  # shape let an empty value flow into assert_current, which then blamed the
+  # wrong thing (`current_pr_excluded`) for a probe failure.
+  RCXC2_SCAN="$TMP/rcxc2-scan"; mkdir -p "$RCXC2_SCAN"
+  ( PATH=/nonexistent-for-gh-probe; export PATH
+    review_consolidate_current_pr "$RCXC2_SCAN" >/dev/null 2>&1 )
+  assert_eq "$?" "2" "RCXCb: an unresolvable invoking PR is refused with rc 2, never returned as the empty string"
+
+  printf 'not-a-number\n' >"$RCXC2_SCAN/current-pr.txt"
+  ( PATH=/nonexistent-for-gh-probe; export PATH
```

Shellcheck (0.11.0)

[warning] PATH is the shell search path. Use another name.

(SC2123)